routemaster/lua/nack.lua

```diff
@@ -0,0 +1,12 @@
+--
+-- Negative-acknowledges a message.
+--
+-- KEYS[1]: list of new message UUIDs
+-- KEYS[2]: zset of pending (nacked) message UUIDs
```

Comment:
Ah, maybe not, looking at the other scripts the arguments are all a bit different anyway, so it's probably fine.